### DISCUSSION TITLE
Is there a way to add an option to launch another executable when starting a game?

### DISCUSSION THREAD

**Discussion by chexo3** [Matoking/protontricks#363] [Q&A]
For instance, I have Hook Line & Sinker installed in Webfishing's prefix, and I want to have the option when I launch webfishing to launch HLS instead, like some games use to pick between different renderers or between 32 and 64 bit.

**Comment by cprn** [comment]
Well, it's been over a year, but normally you'd do such things with Steam's launch options, e.g.:

```sh
protontricks-launch --appid 3146520 ~/path/to/HLS.exe& %command%; pkill HLS.exe
```

This:
1. starts `HLS.exe` in the background using Webfishing's prefix
2. then runs Steam's usual command to start the game,
3. and finally kills the lingering `HLS.exe` background process after you quit the game.

`&` – forks the command (runs in the background)
`;` – waits until the command is done before executing the next one

**Reply by chexo3** [reply]
I think you misunderstood my question

**Reply by cprn** [reply]
Well, after reading it again, I might have. So you didn't want to run something in the background before running the game, you wanted to run an entirely different executable instead of the game? If so, that'd be:

```
protontricks-launch --appid 3146520 ~/path/to/HLS.exe # %command%
```

This runs the executable in Webfishing's prefix, and comments out the Steam command that'd normally start the game (everything after `#` gets ignored).

If it helped, I'm glad, and if it still didn't, well, sorry. Won't bother you again either way.
